### ISSUE TITLE
Problem in Training

### ISSUE THREAD

**Issue by rabiaali95** [bbrattoli/JigsawPuzzlePytorch#6]
The accuracy remains zero forever and the loss value does not change as well.
How to solve this problem?

![Selection_009](https://user-images.githubusercontent.com/31028173/66819878-a84a4900-ef7a-11e9-9c6e-4a23d0f94616.png)

**Comment by bbrattoli** [comment]
This could be due to many different issues. Follow this blog post, maybe it helps http://karpathy.github.io/2019/04/25/recipe/

**Comment by rabiaali95** [comment]
Did you also face the same issue because I am running the same code as the repository?

**Comment by bbrattoli** [comment]
Not that I remember. Try reducing the train set to very few images (10 to 100) and change the parameters (mostly learning rate) until the train accuracy increases. This will make the model overfit since you only have few samples, but at least you will know that the code has no bug and the parameters are correct. This is how I usually start with a new code.

**Comment by rabiaali95** [comment]
I have tried this. Accuracy still remains zero throughout.

**Comment by wangtingwei1993** [comment]
Have you addressed the problem? @rabiaali95

**Comment by rabiaali95** [comment]
increasing the number of cores will solve the problem
